ios/PurchasesHybridCommon/PurchasesHybridCommon/PurchasesAreCompletedBy+HybridAdditions.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PurchasesAreCompletedBy+HybridAdditions.swift
+//  PurchasesHybridCommon
+//
+//  Created by Will Taylor on 7/26/24.
+//  Copyright © 2024 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import RevenueCat
+
+public extension PurchasesAreCompletedBy {
+
+    init?(name: String) {
```

Comment:
I switched the implementation to this over the existing `fromString()` function to match what we're doing in `StoreKitVersion+HybridAdditions` and because it felt a little more "Swift-like" to me - let me know if y'all think we should revert to the previous `fromString()` function!